client/state/stored-cards/reducer.js

```diff
@@ -79,6 +88,32 @@ export const isFetching = ( state = false, action ) => {
 	return state;
 };
 
+/**
+ * `Reducer` function which handles request/response actions
+ * concerning stored card editing
+ *
+ * @param {object} state - current state
+ * @param {object} action - storedCard action
+ * @returns {object} updated state
```

Comment:
You actually know what the type is for this reducer, so we can list it directly. In this case, instead of `{object}` for `state` and the return value, we can put something like `{Record<string, boolean>}`.

For the `action` type, we could be very specific, but it's probably fine to just use `{{type: string, card: {stored_details_id: string}}}`.